BR: refine BR docs (#8628)
pingcap/docs: br/br-usage-restore.md

```

## Restore tables created in the `mysql` schema

> **Warning:**
>
> This is still an experimental feature. It is **NOT** recommended that you use it in the production environment.

BR backs up tables created in the `mysql` schema by default. When you restore data using BR, the tables created in the `mysql` schema are not restored by default. To restore these tables, you can explicitly include them using the [table filter](/table-filter.md#syntax). The following example restores `mysql.usertable` created in the `mysql` schema. The command restores `mysql.usertable` along with other data.

{{< copyable "shell-regular" >}}

```shell
br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
```

In the preceding command,

- `-f '*.*'` is used to override the default rules
- `-f '!mysql.*'` instructs BR not to restore tables in `mysql` unless otherwise stated.
- `-f 'mysql.usertable'` indicates that `mysql.usertable` should be restored.

If you only need to restore `mysql.usertable`, run the following command:

{{< copyable "shell-regular" >}}

```shell
br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
```

> > **Warning:**

Reviewer: 这个 warning 和中文原文不一致。

Author: @hfxsd 中文在这个 PR 里更新过：https://github.com/pingcap/docs-cn/pull/9341/files#diff-4cbbbeff070718d57970da187b47e5190131c705664dedf74be2decfbd10f175R191

>
> Although you can back up system tables (such as `mysql.tidb`) using BR, BR ignores the following system tables even if you use the --filter setting to perform the restoration:
>
> - Statistical information tables (`mysql.stat_*`)
> - System variable tables (`mysql.tidb`, `mysql.global_variables`)
> - User information tables (such as `mysql.user` and `mysql.columns_priv`)
> - [Other system tables](https://github.com/pingcap/tidb/blob/master/br/pkg/restore/systable_restore.go#L31)
>
> Compatibility issues might occur when restoring system tables. Therefore, avoid restoring system tables in production environments.

## Restoration performance and impact

- TiDB fully uses TiKV CPU, disk IO, network bandwidth, and other resources when restoring data. Therefore, it is recommended that you restore backup data on an empty cluster to avoid affecting running services.
- The restoration speed depends much on cluser configuration, deployment, and running services. Generally, the restoration speed can reach 100 MB/s (per TiKV node).

> **Note:**
>

> > The preceding test conclusions, based on simulation tests in many scenarios and verified in some customer sites, have reference value. However, the restoration speed may vary depending on the scenarios. Therefore, you should always run the test and verify the test results.

Reviewer: ```suggestion
> The preceding test conclusions, based on simulation tests in many scenarios and verified in some customer sites, are worthy of reference. However, the restoration speed may vary depending on the scenarios. Therefore, you should always run the test and verify the test results.
```
